src/components/side-panel/interaction-panel/tooltip-config.js

```diff
@@ -68,7 +67,7 @@ const CompareSwitchWrapper = styled.div`
   margin-bottom: 8px;
 `;
 
-function TooltipConfigFactory(DatasetTag) {
+const TooltipConfigFactory = (DatasetTag, FieldSelector) => {
```

Comment:
same here, make it a function instead of `const ...`

Reply:
Done